src/main/resources/migreringer/V1.0_0027__oppgave_v3_statistikk.sql

```diff
@@ -0,0 +1,102 @@
+CREATE TABLE if not exists OMRADE
+(
+    id                          BIGINT GENERATED ALWAYS AS IDENTITY     NOT NULL PRIMARY KEY,
+    ekstern_id                  VARCHAR(100)                            NOT NULL unique
+);
+
+comment on table OMRADE is 'Spesifiserer subdomene for oppgaver';
+comment on column OMRADE.ekstern_id is 'Navn på område, feks "k9"';
+
+CREATE TABLE if not exists FELTDEFINISJON
+(
+    id                          BIGINT GENERATED ALWAYS AS IDENTITY     NOT NULL PRIMARY KEY,
+    ekstern_id                  VARCHAR(100)                            NOT NULL,
+    omrade_id                   BIGINT                                  NOT NULL, --eier
+    liste_type                  boolean                                 NOT NULL,
+    tolkes_som                  VARCHAR(100)                            NOT NULL,
+    --vis_til_bruker              boolean                                 NOT NULL DEFAULT TRUE --ikke aktuelt for statistikk
```

Comment:
skal de kommenterte feltene bort?

Reply:
De er tenkt i bruk når (hvis) dette brukes til oppgavestyring. Det er ikke direkte relevant til statistikkformålet, så det er derfor de er kommentert ut

Reply:
jeg lurer på om det da er bedre å fjerne det herifra fordi det må da lages update table skript for å legge de inn når de en gang kommer på plass. Kanskje putte det et annet sted der de kan være som huske lapp. Kanskje en jira oppgave med ting som dere vet må gjøres en gang senere, evt. en TODO i repository -